src/autopas/containers/linkedCells/LinkedCellsReferences.h

```diff
@@ -292,6 +292,11 @@ class LinkedCellsReferences : public CellBasedParticleContainer<ReferenceParticl
     }
 
     std::vector<ParticleType> invalidParticles;
+
+    // for exception handling in parallel region
+    bool exceptionCatched{false};
```

Comment:
From the top of my head, I have no better idea. Except for renaming the variable to `exceptionCaught` ;)
```suggestion
    // OpenMP regions need to handle exceptions directly, so we need to remember if
    // an exception occurred.
    bool exceptionCaught{false};
```

Reply:
done in [3bac09c](https://github.com/AutoPas/AutoPas/pull/794/commits/3bac09ce3ea6dc898f2b09df090624a9c3a9ce10)